.github/workflows/prod-sync.yml

```diff
@@ -0,0 +1,44 @@
+name: Sync ODFE artifacts to Prod
+
+# For validation the client_payload can be empty
+# Make sure to pass the action as client_payload in the dispatch event for Prod Sync
+# Example: client_payload: { "action": "prod-sync"}
+
+on: 
+  repository_dispatch:
+    types: [prod-sync]
+
+jobs:
+  Sync-Prod:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_STAGING_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_STAGING_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Install python packages
+        run: |
+          sudo pip install boto3
+          sudo pip install pyyaml
+          sudo pip install requests
+          sudo pip install tabulate
+      - name: Moving artifacts from staging to prod
+        run: |
+          #!/bin/bash
+          ARTIFACT_URL="artifacts.opendistroforelasticsearch.amazon.com"
+          cd release-tools/scripts
+          action=${{github.event.client_payload.branch}}
+          if [ -z "$action" ]
+          then
+            ./prod-sync-artifacts.py 
+          else
+            ./prod-sync-artifacts.py $action |tee sync_info.txt
+            mv sync_info.txt sync_info_$(date +%F-%H:%M).txt
+            aws s3 cp sync_info* s3://$ARTIFACT_URL/prod-sync-tracker/
+          fi
+          aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/tarball/*"
+          aws cloudfront create-invalidation --distribution-id E1VG5HMIWI4SA2 --paths "/downloads/odfe-windows/*"
```

Comment:
This wont create invalidation on all folders. Maybe change to just `"/"`

Reply:
The old Prod Sync include invalidation only for these paths. So, I included the same.

Reply:
I prefer /tartball/* and /downloads/*